src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -1687,8 +1687,6 @@ public int ReceiveFrom(byte[] buffer, int offset, int size, SocketFlags socketFl
             ValidateBufferArguments(buffer, offset, size);
             ValidateReceiveFromEndpointAndState(remoteEP, nameof(remoteEP));
 
-            SocketPal.CheckDualModeReceiveSupport(this);
```

Comment:
Can we rename this to `CheckDualModePacketInfoSupport`?

Reply:
updated